index.js

```diff
@@ -100,14 +141,19 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
 
-// console.log(myMap([1,2,3,4,5], triple)) 
-// // => [3,6,9,12,15]
+function triple() {
+    array * 3
+}
 
-// console.log(myMap([2,4,6,8], triple))   
-// // => [6,12,18,24]
-// console.log("%c----------", "color: red") 
+
+console.log(myMap([1,2,3,4,5], triple)) 
```

Comment:
If you're struggling with callbacks, I would take a look at this line and think about how the `triple` function is being passed in!